src/main/java/com/feedzai/commons/sql/abstraction/dml/Query.java

```diff
@@ -68,6 +68,13 @@
      */
     private boolean distinct = false;
 
+    /**
+     * Signals if query should include FOR UPDATE.
+     *
+     * @since ```feedzai.next.release```
```

Comment:
I thought that we didn't use these kind of tags here.

Reply:
We don't. We need to materialize the version manually.

Reply:
+1, this needs to have the next version "hardcoded"